itests/dummydeal_test.go

```diff
@@ -147,9 +147,10 @@ func TestDummydeal(t *testing.T) {
 	require.Equal(t, "cannot accept piece of size 2254421, on top of already allocated 2254421 bytes, because it would exceed max staging area size 4000000", res2.Reason)
 	log.Debugw("got response from MarketDummyDeal for failing deal", "res2", spew.Sdump(res2))
 
-	// Wait for the deal to be added to a sector
+	// Wait for the deal to be added to a sector and be cleanedup so space is made
 	err = f.waitForDealAddedToSector(dealUuid)
 	require.NoError(t, err)
+	time.Sleep(100 * time.Millisecond)
```

Comment:
@dirkmc @nonsense We need this so the previous deal gets time be to cleanedup and space to be released which is done _after_ the deal finishes and we only wait for deal to finish here.

Reply:
This is why the itest was flaky at times as it sent the next deal before this one was cleanedup and so the next deal ran into not enough space issues.